src/Categories/Category/Instance/Simplex.agda

```diff
@@ -32,3 +32,38 @@ open import Relation.Binary.PropositionalEquality
     }
   ; ∘-resp-≈  = λ {_ _ _ f g h i} eq₁ eq₂ x → trans (cong (λ t → proj₁ f t) (eq₂ x)) (eq₁ (proj₁ i x))
   }
+
+open Category Δ
+
+--------------------------------------------------------------------------------
+-- Face + Degeneracy Maps
+
+face-map : ∀ {n} → Fin (ℕ.suc n) → Fin n → Fin (ℕ.suc n)
+face-map Fin.zero k = Fin.suc k
+face-map (Fin.suc i) Fin.zero = Fin.zero
+face-map (Fin.suc i) (Fin.suc k) = Fin.suc (face-map i k)
+
+face-mono : ∀ {n} → (i : Fin (ℕ.suc n)) → _≤_ =[ face-map i ]⇒ _≤_
+face-mono Fin.zero {x} {y} le = s≤s le
+face-mono (Fin.suc i) {Fin.zero} {y} le = z≤n
+face-mono (Fin.suc i) {Fin.suc x} {Fin.suc y} (s≤s le) = s≤s (face-mono i le)
+
+face : ∀ {n} → Fin (ℕ.suc n) → n ⇒ ℕ.suc n
+face i = face-map i , face-mono i
+
+degeneracy-map : ∀ {n} → Fin n → Fin (ℕ.suc n) → Fin n
+degeneracy-map Fin.zero Fin.zero = Fin.zero
+degeneracy-map Fin.zero (Fin.suc Fin.zero) = Fin.zero
```

Comment:
Do you need to split these two (line 56 and 57) ? Looks like you don't.

Reply:
Looks like I can, nice catch :)